README.md

```diff
@@ -1,3 +1,15 @@
+# ask-ai
+The system can answer 2 types of questions:
+
+1. about entity
+2. about subdividing
+
+What can be improved:
+
+1. make ask-ai multilingual, now it is russian 
+2. make llm promt better (sometimes its answers are bad)
```

Comment:
LLM is an acronym, so it's better to be in uppercase

also possibly we can do more than prompt changes here? possibly it's not worth rto specify and we can replace it with smth more general: "improve work LLM"